### PR TITLE
formatting standardization 

### LINKED ISSUE #113: standardization with respect new agreement on ```if, try,for``` statements
--------
Issue Description
--------
Considering the new agreement about inline ```if, try and for``` statements, we need to parse the all code in order to be compliant. 

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [X] 1. Is it tagged with a type: defect or improvement? ***improvement***
- [X] 2. Is it tagged with a priority: critical, normal or minor? ***normal***
- [X] 3. If it will impact requirements or requirements tests, is it tagged with requirements? ***n/a***
- [X] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [X] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.) ***n/a***

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [ ] 1. If the issue is a defect, is the defect fixed?
- [ ] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [ ] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [ ] 4. If the issue is a defect, does it impact the latest stable branch? If yes, is there any issue tagged with stable (create if needed)?
- [ ] 5. If the issue is being closed without a merge request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by alfoa**
--------
Pull Requeset Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)

#113

##### What are the significant changes in functionality due to this change request?
This is a PR to standardize the code for indentation and ```if, try,except,for statements```

Closes #113
----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: framework/CodeInterfaces/OpenModelica/OpenModelicaInterface.py [hunk 1]
```diff
@@ -117,7 +117,8 @@
 Example of multiple parameter override (option 3 above): BouncingBall.exe -override "h=7,g=7,v=2"
 
 To use RAVEN, we need to be able to perturb the input and output files from the defaults.  The command line
-form of this is: (Where the output file will be of the type originally configured)
+form of this is:
+  (Where the output file will be of the type originally configured)
```
Comment (PaulTalbot-INL) [inline framework/CodeInterfaces/OpenModelica/OpenModelicaInterface.py]: False positive, but doesn't need to be changed unless there are other changes.

File: framework/CodeInterfaces/Utilities/csvUtilities.py [hunk 2]
```diff
@@ -113,9 +120,12 @@ def mergeCsvAndReturnOutput(self, options = {}):
     returnAsDict = False
     variablesToExpandFrom.append('time')
     if options:
-      if "sameKeySuffix" in options.keys()          : sameKeySuffix         = options["sameKeySuffix"]
-      if "variablesToExpandFrom" in options.keys()  : variablesToExpandFrom = options["variablesToExpandFrom"]
-      if "returnAsDict"          in options.keys()  : returnAsDict = bool(options["returnAsDict"])
+      if "sameKeySuffix" in options.keys():
+        sameKeySuffix         = options["sameKeySuffix"]
```
Comment (PaulTalbot-INL) [inline framework/CodeInterfaces/Utilities/csvUtilities.py]: Can we remove this excess whitespace?

File: developer_tools/pylint_check [hunk 3]
```diff
@@ -5,4 +5,4 @@ SCRIPT_DIR=`(cd $SCRIPT_DIRNAME; pwd)`
 
 cd $SCRIPT_DIR/..
 
-pylint --disable=all --enable=missing-docstring --enable=no-absolute-import --enable=old-division --enable=print-statement --enable=file-builtin framework/
+pylint --disable=all --enable=missing-docstring --enable=no-absolute-import --enable=old-division --enable=print-statement --enable=file-builtin --enable=multiple-statements framework/
```
Comment (joshua-cogliati-inl) [inline developer_tools/pylint_check]: Now that these have been removed, they can be checked by pylint.

File: framework/CodeInterfaces/Utilities/csvUtilities.py (hunk 2 above)
Comment (alfoa) [inline framework/CodeInterfaces/Utilities/csvUtilities.py]: I can adapt the script to do that.

File: framework/Csv_loader.py [hunk 4]
```diff
@@ -158,45 +160,58 @@ def __csvLoaderForPoint(self,fileIn,options):
...
+        self.raiseAnError(IOError,'pivotParameter ' +pivotParameter+' has not been found in file '+ str(fileIn) + '!')
     else:
       pivotIndex = self.allFieldNames.index("time") if "time" in self.allFieldNames else None
       # if None...default is 0
-      if pivotIndex == None: pivotIndex = 0
-    if inputRow > data[:,0].size-1  and inputRow != -1: self.raiseAnError(IOError,'inputRow is greater than number of actual rows in file '+ str(fileIn) + '!')
+      if pivotIndex == None:
+        pivotIndex = 0
+    if inputRow > data[:
+      ,0].size-1  and inputRow != -1: self.raiseAnError(IOError,'inputRow is greater than number of actual rows in file '+ str(fileIn) + '!')
```
Comment (joshua-cogliati-inl) [inline framework/Csv_loader.py]: Hm, this is not the best.

File: framework/Csv_loader.py [hunk 5]
```diff
@@ -158,45 +160,58 @@ def __csvLoaderForPoint(self,fileIn,options):
...
+      self.fieldNames = outParam
 
     #fill input param dictionary
     for key in inParam:
       ix = self.allFieldNames.index(key) if key in self.allFieldNames else None
       if ix != None:
         if inputPivotVal != None:
-          if float(inputPivotVal) > np.max(data[:,pivotIndex]) or float(inputPivotVal) < np.min(data[:,pivotIndex]): self.raiseAnError(IOError,'inputPivotVal is out of the min and max for input  ' + key+' in file '+ str(fileIn) + '!')
+          if float(inputPivotVal) > np.max(data[:
+            ,pivotIndex]) or float(inputPivotVal) < np.min(data[:,pivotIndex]): self.raiseAnError(IOError,'inputPivotVal is out of the min and max for input  ' + key+' in file '+ str(fileIn) + '!')
```
Comment (joshua-cogliati-inl) [inline framework/Csv_loader.py]: This also might need manual adjustment, or change the script.

File: framework/Csv_loader.py [hunk 6]
```diff
@@ -207,39 +222,52 @@ def __csvLoaderForPoint(self,fileIn,options):
         for key in outParam:
           if key in self.allFieldNames:
             outDict[key] = np.atleast_1d(np.array(data[lastRow,self.allFieldNames.index(key)]))
-          else: self.raiseAnError(IOError,"the parameter " + str(key) + " has not been found")
+          else:
+            self.raiseAnError(IOError,"the parameter " + str(key) + " has not been found")
     elif outputRow != None:
-      if outputRow > data[:,0].size-1  and outputRow != -1: self.raiseAnError(IOError,'outputRow is greater than number of actual rows in file '+ str(fileIn) + '!')
+      if outputRow > data[:
+        ,0].size-1  and outputRow != -1: self.raiseAnError(IOError,'outputRow is greater than number of actual rows in file '+ str(fileIn) + '!')
```
Comment (joshua-cogliati-inl) [inline framework/Csv_loader.py]: Another fail.

**Comment by joshua-cogliati-inl** [comment]
Is the script available?

File: framework/Csv_loader.py (hunk 6 above)
Comment (joshua-cogliati-inl) [inline framework/Csv_loader.py]: I think all these could be fixed if the script checked if the next non-whitespace character was not a comma.

Comment (PaulTalbot-INL) [inline framework/Csv_loader.py]: Should we cover comma-less things like:
```
if stuff: data[2,:]
else: data[:][1]
```

Comment (joshua-cogliati-inl) [inline framework/Csv_loader.py]: Hm, I haven't seen any, but I stopped about a third of the way through.

File: framework/CodeInterfaces/OpenModelica/OpenModelicaInterface.py (hunk 1 above)
Comment (alfoa) [inline framework/CodeInterfaces/OpenModelica/OpenModelicaInterface.py]: fixed

File: framework/Csv_loader.py (hunk 4 above)
Comment (alfoa) [inline framework/Csv_loader.py]: fixed

File: framework/Csv_loader.py (hunk 5 above)
Comment (alfoa) [inline framework/Csv_loader.py]: fixed

File: framework/Csv_loader.py (hunk 6 above)
Comment (alfoa) [inline framework/Csv_loader.py]: i fixed all of them

**Comment by alfoa** [comment]
addressed comments and removed inconsistencies and artifacts of the script

**Comment by alfoa** [comment]
Paul whenever you want this is ready to be merged (in case you agree on the modifications)

**Pull request merged**